cristin-project/src/test/java/no/unit/nva/cristin/projects/query/QueryCristinProjectHandlerTest.java

```diff
@@ -620,43 +619,44 @@ void handlerThrowsBadRequestWhenStatusQueryParamsIsInvalid() throws IOException
         assertEquals(PROBLEM_JSON, gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
         assertThat(body.getDetail(),
                 containsString(
-                        invalidQueryParametersMessageWithRange(STATUS, Arrays.toString(ProjectStatus.values()))));
+                        ErrorMessages.invalidQueryParametersMessageWithRange(JsonPropertyNames.STATUS,
+                                Arrays.toString(ProjectStatus.values()))));
     }
 
     @ParameterizedTest(
             name = "Handler accepts query parameter status in any case {0}")
     @CsvSource({"ACTIVE", "CONCLUDED", "NOTSTARTED", "active", "concluded", "notstarted", "nOtsTaRtEd"})
     void handlerAcceptsQueryParamsStatusInAnycase(String statusQuery) throws IOException {
         InputStream input = requestWithQueryParameters(Map.of(
-                QUERY, RANDOM_TITLE + WHITESPACE + RANDOM_TITLE,
-                ORGANIZATION, SAMPLE_NVA_ORGANIZATION_ENCODED,
-                STATUS, statusQuery));
+                JsonPropertyNames.QUERY, RANDOM_TITLE + WHITESPACE + RANDOM_TITLE,
+                JsonPropertyNames.ORGANIZATION, SAMPLE_NVA_ORGANIZATION_ENCODED,
+                JsonPropertyNames.STATUS, statusQuery));
 
         handler.handleRequest(input, output, context);
 
         var gatewayResponse =
                 GatewayResponse.fromOutputStream(output, SearchResponse.class);
 
-        assertEquals(HttpURLConnection.HTTP_OK, gatewayResponse.getStatusCode());
+        assertEquals(HTTP_OK, gatewayResponse.getStatusCode());
     }
 
     private void fakeAnEmptyResponseFromQueryAndEnrichment() throws ApiGatewayException {
-        cristinApiClientStub = spy(cristinApiClientStub);
-        doReturn(new HttpResponseFaker(EMPTY_LIST_STRING, HttpURLConnection.HTTP_OK,
+        cristinApiClientStub = Mockito.spy(cristinApiClientStub);
+        Mockito.doReturn(new HttpResponseFaker(EMPTY_LIST_STRING, HTTP_OK,
                 generateHeaders(ZERO_VALUE, LINK_EXAMPLE_VALUE)))
                 .when(cristinApiClientStub).queryProjects(any(), any());
-        doReturn(Collections.emptyList()).when(cristinApiClientStub).fetchQueryResultsOneByOne(any());
+        Mockito.doReturn(Collections.emptyList()).when(cristinApiClientStub).fetchQueryResultsOneByOne(any());
```

Comment:
Why do you not want to use static imports? There is alot of changes in this class just reverting already existing code because of not using static imports. When i first started in this team i was told the same, that using static imports is better. Maybe I need a second opinion from another reviewer.

Reply:
I was just trying to avoid this issue of the * imports.

Reply:
* imports is not used for static imports. a static import can be `import static no.unit.nva.model.Organization.ORGANIZATION_CONTEXT;`